### DISCUSSION TITLE
Polar decomposition

### DISCUSSION THREAD

**Discussion by Cyan2077** [adtzlr/ttb#53] [Ideas]
This toolbox is really nice. I have small suggestion that you can add the ' Polar decomposition' function  into this toolbox. Because when we edit UMAT in abaqus, if we use keyword 'orientation', the deformation tensor F will be passed in UMAT in local coordinate. We always need  'Polar decomposition'  to get the rotation part of the F, so that we can rotate the F back to the global coordinate.

**Comment by adtzlr** [comment]
Hi @Cyan2077 and thank you for your suggestion, great idea. I'll add it to the issues.

**Reply by adtzlr** [reply]
Please see #54 for further details.
